test/e2e-setup/bicep/modules/nodepool.bicep

```diff
@@ -10,6 +10,9 @@ param replicas int = 2
 @description('OpenShift Version ID to use')
 param openshiftVersionId string = '4.19.0'
 
+@description('VM size for the nodepool VMs')
+param vmSize string = 'Standard_D8s_v3'
```

Comment:
rather than add a parameter, how about we create an entirely new file.  The benefit to a new file is that as people write more tests and want more options, we'll never modify what *this* test runs.  That allows us to write a test once and never worry about what changes as we on.  If a test does eventually break or we break an API or instance type, we'll be feeling the same pain as customers.

Reply:
For this particular test (changing `vmSize` only), updating the parameter here is completely reasonable. I suggest against such duplication here.